cmd/telegram-butler/setup.go

```diff
@@ -11,31 +12,40 @@ import (
 	"github.com/GolangUA/telegram-butler/internal/handler/join"
 	"github.com/GolangUA/telegram-butler/internal/handler/message"
 	"github.com/GolangUA/telegram-butler/internal/handler/message/commands"
-	"github.com/GolangUA/telegram-butler/internal/module/logger"
 	"github.com/GolangUA/telegram-butler/internal/module/telegram"
 )
 
-func setup(ctx context.Context, log logger.Logger) (run func() error, stop func() error, err error) {
+func setup(ctx context.Context, log *slog.Logger) (run func() error, stop func() error, err error) {
+	log.Info("Setting up the Bot")
+
 	botCfg := config.Bot()
 	bot, err := telegram.Bot(botCfg)
 	if err != nil {
 		return nil, nil, fmt.Errorf("bot: %w", err)
 	}
 
+	log.Debug("Bot is set")
+
 	if err = commands.Sync(bot); err != nil {
 		return nil, nil, fmt.Errorf("sync commands failed: %w", err)
 	}
 
+	log.Debug("Commands are synced")
+
 	webhookCfg, err := config.Webhook()
 	if err != nil {
 		return nil, nil, fmt.Errorf("webhook config: %w", err)
 	}
 
+	log.Debug("Webhook is configured", slog.Any("webhook", webhookCfg))
```

Comment:
I tried to hide some fields from this config by using [`LogValuer`](https://pkg.go.dev/log/slog#LogValuer), but my tries were not successful

Can someone checked this idea to hide some props?
Or we can just leave this as it is now with `Debug` level?

Reply:
Update: it's actually works, but not with custom handler, it needs to be updated (but do we need?)

Reply:
@andrsj let's than print just a host as a string value instead of the whole webhook config

Reply:
# `LOG_FORMAT`

@vpakhuchyi 

## `prettyjson`

![image](https://github.com/GolangUA/telegram-butler/assets/61803449/c179ab09-c3ce-482c-a85f-bc1b62a5bb76)

## `text`

```log
time=2024-04-11T18:26:52.919+03:00 level=DEBUG msg="Webhook is configured" webhook.bot_token=[REDACTED] webhook.webhook_host=bb95-46-173-102-34.ngrok-free.app
```

## `json`

```json
{"time":"2024-04-11T18:27:49.962494789+03:00","level":"DEBUG","msg":"Webhook is configured","webhook":{"bot_token":"[REDACTED]","webhook_host":"7ee3-46-173-102-34.ngrok-free.app"}}
```